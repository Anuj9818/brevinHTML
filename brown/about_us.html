<!DOCTYPE html>
<html lang="en">
<head>
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1">
<meta name="description" content="">
<meta name="author" content="">
<link rel="shortcut icon" href="images/favicon.png" type="image/x-icon">
<title>Contact us | About us</title>
<link rel="stylesheet" type="text/css" href="css/bootstrap.css" />
<link rel="stylesheet" type="text/css" href="font-awesome-4.7.0/css/font-awesome-4.7.0.min.css">
<link rel="stylesheet" type="text/css" href="css/style.css" />
<link rel="stylesheet" type="text/css" href="css/responsive.css" />

<!-- HTML5 shim and Respond.js for IE8 support of HTML5 elements and media queries -->
<!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
<!--[if lt IE 9]>
      <script src="https://oss.maxcdn.com/html5shiv/3.7.2/html5shiv.min.js"></script>
      <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
    <![endif]-->

<link href="https://fonts.googleapis.com/css?family=Josefin+Sans:400,600,700" rel="stylesheet">
<link href="https://fonts.googleapis.com/css?family=Lato:400,400i,700,700i" rel="stylesheet">
</head>

<body>
<div id="billboard">
  <header class="header"><!--//.header starts-->
    <div id="pav-mainnav">
      <div class="navigation-bar">
        <div class="container">
          <div class="navbar" role="navigation">
            <div class="navbar-header navbar-inverse">
              <figure class="logo_holder"> <a href="index.html"><img src="images/main-logo.png"></a> </figure>
              <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar"></span> <span class="icon-bar"></span> <span class="icon-bar"></span> </button>
            </div>
            <!--navbar-inverse-->
            <div class="collapse navbar-collapse">
              <ul class="nav navbar-nav">
                <li><a href="index.html">Home </a></li>
                <li class="active"><a href="about_us.html"> About Us </a></li>
                <li><a href="services.html"> Services </a> </li>
                <li><a href="contact_us.html" > Contact Us</a></li>
              </ul>
              <!--navbar-nav--> 
            </div>
            <!-- /.nav-collapse --> 
          </div>
          <!--navbar--> 
        </div>
      </div>
      <!--navigation-bar--> 
    </div>
    <!--pav-mainnav--> 
    
  </header>
  <!--//.header ends--> 
  
</div>
<!--billboard ends--> 

<!--.//.MAIN SLIDER STARTS-->

<section class="sub_page-banner">
  <figure class="full_banner"> <img src="images/welcome_banner.jpg" alt="" title=""> </figure>
  <div class="breadCrumbNav">
    <div class="container">
      <h2 class="page_title"> About us</h2>
    </div>
  </div>
</section>
<!--.//.MAIN SLIDER ENDS--> 

<!--//CONTACT US CONTENT STARTS-->
<section class="welcome_content wrapper">
  <div class="container inner_content">
    <figure class="thumbnail_image flt_rht"> <img src="images/welcome_banner.jpg" alt="" title=""> </figure>
    <div class="txt_wrapper">
      <h1>Introduction to Brown Beauty!</h1>
      <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries.
        It is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout. The point of using Lorem Ipsum is that it has a more-or-less normal distribution of letters, as opposed to using 'Content here, content here', making it look like readable English. Many desktop publishing packages and web page editors now use Lorem Ipsum as their default model text, and a search for 'lorem ipsum' will uncover many web sites still in their infancy. 
        
        It is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout. The point of using Lorem Ipsum is that it has a more-or-less normal distribution of letters, as opposed to using 'Content here, content here', making it look like readable English. Many desktop publishing packages and web page editors now use Lorem Ipsum as their default model text, and a search for 'lorem ipsum' will uncover many web sites still in their infancy. </p>
      <p>It is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout. The point of using Lorem Ipsum is that it has a more-or-less normal distribution of letters, as opposed to using 'Content here, content here', making it look like readable English. Many desktop publishing packages and web page editors now use Lorem Ipsum as their default model text, and a search for 'lorem ipsum' will uncover many web sites still in their infancy. It is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout. The point of using Lorem Ipsum is that it has a more-or-less normal distribution of letters, as opposed to using 'Content here, content here', making it look like readable English. Many desktop publishing packages and web page editors now use Lorem Ipsum as their default model text, and a search for 'lorem ipsum' will uncover many web sites still in their infancy. It is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout. The point of using Lorem Ipsum is that it has a more-or-less normal distribution of letters, as opposed to using 'Content here, content here', making it look like readable English. Many desktop publishing packages and web page editors now use Lorem Ipsum as their default model text, and a search for 'lorem ipsum' will uncover many web sites still in their infancy. </p>
    </div>
  </div>
</section>

<!--//CONTACT US CONTENT ENDS-->

<footer class="footer wrapper">
  <div class="container">
    <h2>Contact us</h2>
    <div class="row inner_content">
      <div class="col-sm-4 col-xs-12">
        <address>
        <figure class="icon"> <i class="fa fa-map-marker" aria-hidden="true"></i> </figure>
        <div class="details"> <span> your address </span> </div>
        </address>
      </div>
      <div class="col-sm-4 col-xs-12">
        <address>
        <figure class="icon"> <i class="fa fa-phone" aria-hidden="true"></i> </figure>
        <div class="details"> <span> <a href="tel:01-5555039" class="linkto">01-5555039 </a> <a href="tel:01-98124578" class="linkto">01-98124578</a></span> </div>
        </address>
      </div>
      <div class="col-sm-4 col-xs-12">
        <address>
        <figure class="icon"> <i class="fa fa-envelope" aria-hidden="true"></i> </figure>
        <div class="details"> <span> <a href="mailto:info@brownbeauty.com" class="linkto">info@brownbeauty.com</a> </span> </div>
        </address>
      </div>
      <div class="col-xs-12">
        <div class="social-icons">
          <ul>
            <li><a href="#" data-toggle="tooltip" data-placement="top" title="" data-original-title="Facebook"><i class="fa fa-facebook" aria-hidden="true"></i> </a></li>
            <li><a href="#" data-toggle="tooltip" data-placement="top" title="" data-original-title="Twitter"><i class="fa fa-twitter" aria-hidden="true"></i> </a></li>
            <li><a href="#" data-toggle="tooltip" data-placement="top" title="" data-original-title="Linkedin"><i class="fa fa-linkedin" aria-hidden="true"> </i></a></li>
            <li><a href="#" data-toggle="tooltip" data-placement="top" title="" data-original-title="Google-plus"><i class="fa fa-google-plus" aria-hidden="true"></i> </a></li>
          </ul>
        </div>
      </div>
    </div>
  </div>
</footer>
<section class="copyright">
  <div class="container">
    <p>The Brown Beauty lounge Â©2019. All Rights Reserved</p>
  </div>
</section>
<section class="back-top">
  <div id="back-top" class="uparrow" style="display: block;"> <a href="#top" title="Go to the top of the page"> <i class="fa fa-angle-up"></i> </a> </div>
</section>
</body>
<script src="js/jquery-1.9.1.min.js"></script>
<script src="js/fixed-nav.js"></script>
<script src="js/jquery.js"></script>
<script src="js/bootstrap.js"></script>
<script src="js/Push-up.js"></script>
<script src="js/Jquery simple slider.js"></script>

<!--..//SOCIAL ICON HOVER TOOLTIP STARTS-->
<script>
$(document).ready(function(){
    $('[data-toggle="tooltip"]').tooltip();   
});
</script>
</html>
