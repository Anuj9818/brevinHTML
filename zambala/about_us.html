<!DOCTYPE html>
<html lang="en">
<head>
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1">
<meta name="description" content="">
<meta name="author" content="">
<meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">
<link rel="shortcut icon" href="images/favicon.png" type="image/x-icon">
<title>Zambla |  About us</title>
<link rel="stylesheet" type="text/css" href="css/bootstrap.css" />
<link rel="stylesheet" type="text/css" href="font-awesome-4.7.0/css/font-awesome-4.7.0.min.css">
<link rel="stylesheet" type="text/css" href="css/animate.css" />
<link rel="stylesheet" type="text/css" href="css/owl.carousel.css"/>
<link  rel="stylesheet" type="text/css" href="SmartDroddownMenus/Dropdownjquery.css">
<link rel="stylesheet" type="text/css" href="css/simple_slider.css"/>
<link rel="stylesheet" type="text/css"  href="css/default.css"/>
<link rel="stylesheet" type="text/css" href="css/navbar.css"/>
<link rel="stylesheet" type="text/css" href="css/style.css" />
<link rel="stylesheet" type="text/css" href="css/responsive.css" />
<link rel="stylesheet" type="text/css" href="css/side_nav.css" />
<link rel="stylesheet" type="text/css" href="css/programmer.css" />
<link href="https://fonts.googleapis.com/css?family=Lora:400,400i,700,700i" rel="stylesheet">
<link href="https://fonts.googleapis.com/css?family=Lato:400,400i,700,700i" rel="stylesheet">

<!-- HTML5 shim and Respond.js for IE8 support of HTML5 elements and media queries -->
<!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
<!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.2/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->

</head>
<body>
<header class="header"><!--../header starts-->
  <div id="pav-mainnav"><!-- ./pav-mainnav-->
    <div class="navigation-bar"><!-- ./navigation-bar-->
      <div class="container">
        <div class="row">
          <div class="col col-sm-3 col-xs-12">
            <figure class="logo_holder"><a href="index.html"> <img src="images/main_logo.png"> </a></figure>
          </div>
          <div class="col col-sm-9 col-xs-12">
            <div id="mainContent">
              <div id="myCanvasNav" class="overlay3" onclick="closeOffcanvas()" style="width: 0%; opacity: 0;"></div>
              <div id="myOffcanvas" class="offcanvas" >
                <div class="navbar navbar-default" role="navigation"> <!--//Navbar -->
                  <div class="navbar-header"> </div>
                  <div class="side_nav"><!--nav-collapse-->
                    <ul class="nav navbar-nav">
                      <li><a href="index.html"> Home </a> </li>
                      <li class="active"><a href="about_us.html"> About Us </a> </li>
                      <li><a href="rooms.html"> Rooms <span class="caret"></span> </a>
                        <ul class="dropdown-menu animated fadeInDown">
                          <li><a href="#">dropdown1</a></li>
                          <li><a href="#">dropdown2 </a></li>
                          <li><a href="#">dropdown3</a></li>
                        </ul>
                      </li>
                      <li><a href="facilities.html">Facilities </a></li>
                      <li><a href="attraction.html"> Attraction<span class="caret"></span></a>
                        <ul class="dropdown-menu animated fadeInDown">
                          <li><a href="#">dropdown1</a></li>
                          <li><a href="#">dropdown2 </a></li>
                          <li><a href="#">dropdown3</a></li>
                        </ul>
                      </li>
                      <li><a href="gallery_listing.html"> Photo </a></li>
                      <li><a href="gallery_listing.html"> Video </a></li>
                      <li><a href="contact_us.html"> contact us </a> </li>
                    </ul>
                  </div>
                  <!--/.nav-collapse --> 
                </div>
                <!--//Navbar --> 
              </div>
              <!--offcanvas-->
              <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse"  onclick="openNav3();openOffcanvas()"> <span class="toggle-bar"></span> </button>
            </div>
          </div>
        </div>
      </div>
    </div>
    <!-- ./navigation-bar--> 
  </div>
  <!-- ./pav-mainnav--> 
</header>
<!--../header ends-->

<section class="sub_page-banner">
  <figure class="full_banner"> <img src="images/banner1.jpg" alt="" title=""> </figure>
  <div class="breadCrumbNav">
    <div class="container">
      <h1 class="page_title"> About us</h1>
      <h4 class="caption">A Landmark Resort in Halesi</h4>
    </div>
  </div>
</section>
<section class="subpage_content about_page wrapper">
  <div class="container inner_content">
    <h2 class="title center">Welcome To Zambla Resort</h2>
  </div>
  <div class="z-fullimage">
    <div class="container">
      <div class="row">
        <div class="z_col col col-sm-offset-4 col-sm-4">
          <div class="txt_wrapper">
            <h3 class="sub_title">A Landmark Resort in Halesi</h3>
            <p>Located at Halesi, Khotang: peaceful, religious and holy place, Zambala Resort has a capability to provide full satisfaction to its customers with attractive infrastructure, Family environment and Professional staffs. At Zambala Resort, Guest can find the best customer service in whole Halesi Village.
            <p> Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et quasi architecto beatae vitae dicta sunt explicabo.  Nemo enim ipsam voluptatem quia voluptas sit aspernatur aut odit aut fugit, sed quia consequuntur magni dolores eos qui ratione voluptatem sequi nesciunt. Neque porro quisquam est, qui dolorem ipsum quia dolor sit amet</p>
            <h4>Zambala</h4>
            <p>Zambala simply refers to the Holy God of wealth in Tibetan language.</p>
            <b>Our Motto</b>
            <h4>COME AS GUEST GO AS A FRIEND</h4>
            <p>Zambala welcomes all tourists as a guest. It will serve you the best quality of service in friendly way. And when you will leave it will say good bye and send you as a friend of Zambala.</p>
          </div>
        </div>
      </div>
    </div>
  </div>
</section>

<!--// footer starts-->
<footer class="footer wrapper">
  <div class="container wrapper">
    <div class="row">
      <div class="col col-sm-3 col-xs-4">
        <h4 class="footer_title">Quick Link </h4>
        <ul>
          <li><a href="index.html">Home </a></li>
          <li><a href="about_us.html">About Us </a></li>
          <li><a href="rooms.html">Rooms </a></li>
          <li><a href="facilities.html">Facilities </a></li>
          <li><a href="attraction.html">Attraction </a></li>
          <li><a href="gallery_listing.html">Photo </a></li>
          <li><a href="gallery_listing.html">Video</a></li>
          <li><a href="contact_us.html">Contact Us</a></li>
        </ul>
      </div>
      <div class="col col-sm-4 col-xs-8">
        <h4 class="footer_title">Latest News</h4>
        <ul>
          <li> Grand opening of Zambala Resort by Sangye Nyenpa Rinpoche <span class="time">Date: Feb 02, 2017</span></li>
          <li> Grand opening of Zambala Resort by Sangye Nyenpa Rinpoche <span class="time">Date: Feb 02, 2017</span></li>
          <li> Grand opening of Zambala Resort by Sangye Nyenpa Rinpoche <span class="time">Date: Feb 02, 2017</span></li>
        </ul>
      </div>
      <div class="col col-sm-3 col-xs-6">
        <h4 class="footer_title">Contacts Us</h4>
        <ul>
          <li>
            <figure class="icon"> <i class="fa fa-map-marker" aria-hidden="true"></i> </figure>
            <div class="details"> <span>Halesi (Maratika), Khotang</span> </div>
          </li>
          <li>
            <figure class="icon"><i class="fa fa-volume-control-phone" aria-hidden="true"></i></figure>
            <div class="details"> <span><a href="tel:036410084">036410084</a></span> </div>
          </li>
          <li>
            <figure class="icon"><i class="fa fa-envelope"></i> </figure>
            <div class="details"> <a href="mailto:info@zambalahalesi.com">Email:info@zambalahalesi.com</a> </div>
          </li>
        </ul>
      </div>
      <div class="col col-sm-2 col-xs-6">
        <h4 class="footer_title">Follows Us</h4>
        <div class="social-icons">
          <ul>
            <li><a href="#"><i class="fa fa-facebook" aria-hidden="true"></i> </a></li>
            <li><a href="#"><i class="fa fa-twitter" aria-hidden="true"></i> </a></li>
            <li><a href="#"><i class="fa fa-youtube" aria-hidden="true"></i></a></li>
          </ul>
        </div>
      </div>
    </div>
  </div>
  <div class="copyright">
    <div class="container">
      <p class="flt_lft">Copyright Â© 2019 Zambla Resort.All Rights Reserved</p>
      <p class="flt_rht">Website by Brevin Creation</p>
    </div>
  </div>
</footer>

<!--// footer ends-->

<section class="back_top"><!--//back to top scroll-->
  <div class="container">
    <div id="back-top" style="display: block;"> <a href="#top" title="Go to top"><i class="fa fa-angle-up" aria-hidden="true"></i> </a> </div>
  </div>
</section>
<!--//back to top scroll--> 

<script src="js/jquery-1.9.1.min.js"></script> 
<script src="js/fixed-nav.js"></script> 
<script src="js/jquery.js"></script> 
<script src="js/bootstrap.js"></script> 
<script src="js/Push up jquery.js"></script> 

<!-- SmartMenus DroddownMenujQuery starts --> 
<script type="text/javascript" src="SmartDroddownMenus/Dropdownjquery_002.js"></script> 
<script type="text/javascript" src="SmartDroddownMenus/Dropdownjquery.js"></script> 
<!-- SmartMenus DroddownMenujQuery ends --> 

<!-- ..// ANIMATABLE JQERY STARTS //--> 
<script type="text/javascript" src="js/annimatable_jquery.js"></script> 
<!-- ../SIDENAV JQUERY STARTS//--> 
<script type="text/javascript" src="js/side_nav.js"></script>
</body>
</html>
