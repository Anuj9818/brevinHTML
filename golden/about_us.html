<!DOCTYPE html>
<html lang="en">
<head>
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1">
<meta name="description" content="">
<meta name="author" content="">
<link rel="shortcut icon" href="images/favicon.ico" type="image/x-icon">
<title>Home | About us</title>
<link rel="stylesheet" type="text/css" href="css/bootstrap.css" />
<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.2.0/css/all.css" integrity="sha384-hWVjflwFxL6sNzntih27bfxkr27PmbbK/iSvJ+a4+0owXq79v+lsFkW54bOGbiDQ" crossorigin="anonymous">
<link rel="stylesheet" type="text/css" href="FontAwesome 4.1.0/css/font-awesome.css"/>
<link rel="stylesheet" type="text/css" href="css/style.css" />
<link rel="stylesheet" type="text/css" href="css/responsive.css" />
<link href="css/owl.carousel.css" rel="stylesheet">
<link href="css/jquery.bxslider.css" rel="stylesheet">

<!-- HTML5 shim and Respond.js for IE8 support of HTML5 elements and media queries -->
<!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
<!--[if lt IE 9]>
      <script src="https://oss.maxcdn.com/html5shiv/3.7.2/html5shiv.min.js"></script>
      <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
    <![endif]-->

<!--logo slider Starts -->

<!--logo slider ends -->
<link href="https://fonts.googleapis.com/css?family=Open+Sans:300,300i,400,400i,600,600i,700,700i,800,800i" rel="stylesheet">
<link href="https://fonts.googleapis.com/css?family=Montserrat:300,400,500,600,700,800" rel="stylesheet">
<link href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700" rel="stylesheet">
</head>

<body>
 
  <header class="header"><!--//.header starts-->
    <div class="top-bar">
      <div class="container">
        <ul class="social-icons">
          <li><a href="#"><i class="fa fa-facebook" aria-hidden="true"></i> </a></li>
          <li><a href="#"><i class="fa fa-twitter" aria-hidden="true"></i> </a></li>
          <li><a href="#"><i class="fa fa-linkedin" aria-hidden="true"> </i></a></li>
          <li><a href="#"><i class="fa fa-google-plus" aria-hidden="true"></i> </a></li>
        </ul>
      </div>
    </div>
    <div id="pav-mainnav">
      <div class="navigation-bar">
        <div class="container">
          <div class="row">
            <div class="col-sm-4">
              <figure class="logo_holder"> <a href="index.html"><img src="images/main-logo.png"></a> </figure>
            </div>
            <div class="col-sm-8 nav_col">
              <div class="navbar" role="navigation"> 
                <!--navbar-->
                <div class="navbar-header navbar-inverse">
                  <h4>Menu</h4>
                  <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar"></span> <span class="icon-bar"></span> <span class="icon-bar"></span> </button>
                </div>
                <div class="collapse navbar-collapse"><!--nav-collapse-->
                  
                  <ul class="nav navbar-nav">
                    <li><a href="index.html">Home </a></li>
                    <li class="active"><a href="about_us.html" > About Us </a></li>
                    <li><a href="our_services.html"> Our Services </a> </li>
                    <li><a href="our_clients.html"> Our Clients </a></li>
                    <li><a href="gallery.html"> Gallery</a></li>
                    <li><a href="contact_us.html" > Contact Us</a></li>
                  </ul>
                </div>
              </div>
            </div>
            <!-- /.nav-collapse --> 
          </div>
        </div>
      </div>
    </div>
  </header>
  <!--//.header ends--> 
  
 

<!--.//.MAIN SLIDER STARTS-->
<section class="banner">

    <figure class="about-banner">
     <img src="images/about_us-banner.jpg"  alt="" title=""> 
     </figure>
<div class="breadCrumbNav">
<div class="container">
<h2 class="page_title">About Us</h2>
</div>

</div>
</section>
<!--.//.MAIN SLIDER ENDS--> 


 

<!--//ABOUT US CONTENT STARTS-->
<section class="content-holder about_content page_holder">
  <div class="container">
    <div class="inner_content">
      <div class="txt_wrapper">
        <h3>Quality cleaning</h3>
        <p>We are accountable, Professional, Experienced and 
          fully-trained. With office in Satdobato Lalitpur , Golden 
          Triangle offers a Total Facilities Management Service and solution to fulfill your need to deliver well-managed, clean, safe, secure and efficient healthy premises for you and your staff. We provide intelligent, reliable and m
          easured facilities management services.</p>
      </div>
      <figure class="main_fig"> <img src="images/about-1.jpg" alt="" title="">
        <h4>Your satisfaction our desire </h4>
      </figure>
    </div>
    
    <div class="list-item">
      <h4>Why goldentriangle housekeepers?</h4>
      <ul>
        <li>We know you have better things to do with your time then spending your weekends cleaning. Let's us do it for you!</li>
        <li>We match housekeepers and homeowners.
         We listen to each individual client's needs and then make the appropriate match using the list of services offered by each housekeeper.</li>
        <li>For your convenience we accept Visa, MasterCard and Discover</li>
        <li>We do background checks on all of our experienced houskeepers.</li>
      </ul>
    </div>
    
    <div class="inner_content">
      <div class="txt_wrapper">
        <h3>Professional staff</h3>
        <p>Well trained staff Provider of professional house cleaning servicesGolden Triangle housekeeping and Cleaning Service is a professional Vancouver and Richmond house cleaning service that is licensed, insured and accredited from the Better Business Bureau. We provide a wide range of Vancouver and Richmond quality house cleaning services that include housekeeping and maid services, move out cleaning, move in cleaning, top to bottom open house cleaning packages, carpet cleaning and post construction cleaning for apartments, condos and houses in Vancouver and the Lower Mainland..</p>
      </div>
      <figure class="main_fig"> <img src="images/about-2.jpg" alt="" title="">
        <h4>Handle with care</h4>
      </figure>
    </div>
    
    <div class="list-item">
      <h4>Why choose our golden triangle cleaning service?</h4>
      <ul>
        <li>Our maids out shine and go a step further than other house cleaning and maid services and our cleaning teams are highly trained and have 10 + years of residential cleaning experience.</li>
        <li>Our cleaning service teams are thorough and pay attention to the smaller details making us a hit with our clients.</li>
        <li>We move (light or easy to move) furniture to clean hard to reach areas, clean baseboards, under beds, and we go the extra mile to ensure that our clients are left completely satisfied, We are the Vancouver and Richmond cleaning service to call when you want reliable and thorough cleaning.</li>
 
      </ul>
    </div>
    
    
    
    
    
    <div class="inner_content">
      <div class="txt_wrapper">
        <h3>Affordable rates</h3>
        <p>Affordable cleaning service for all House CleaningWe have been cleaning homes and satisfying customers for many years. golden triangle Housekeepers is dedicated to providing dependable, professional maid service at affordable rates. We offer a wide variety of cleaning tasks to not only meet your needs, but to meet your budget as well. With some house cleaning services the cleaners are all working on a schedule that has them finishing at the same time so they can quickly move on to the next house.</p>
      </div>
      <figure class="main_fig"> <img src="images/about-3.jpg" alt="" title="">
        <h4>Cleaning task meet your budget</h4>
      </figure>
    </div>
    
    <div class="list-item">
      <h4>Tell us what you want and we'll do our best to meet your needs ?</h4>
      <ul>
        <li>All of our housekeepers are professionals. They will treat your home as if it were their own. </li>
        <li>You no longer have to worry about teams of cleaners coming into your home while you're at work.</li>
        <li>We assign one housekeeper to take care of all of your cleaning needs. Not us! If you need your refrigerator cleaned, or some other chores done, we will gladly take care of it. We'll even can do interior windows!.</li>
 
      </ul>
    </div>
    
  </div>
</section>
<!--//ABOUT US CONTENT ENDS--> 
 
<section class="content-holder footer">
  <div class="container">
    <div class="row inner_content">
      <div class="col-sm-3 col-xs-12">
        <h4 class="title">Info</h4>
        <ul class="page_links sameheight3">
          <li><a href="index.html">Home </a></li>
          <li><a href="about_us.html">About Us </a></li>
          <li><a href="our_services.html">Our Services </a></li>
          <li><a href="our_clients.html">Our Clients </a></li>
          <li><a href="news.html">News & Updates </a></li>
          <li><a href="gallery.html">GAllery </a></li>
          <li><a href="contact_us.html">Contact US </a></li>
        </ul>
      </div>
      <div class="col-md-offset-1 col-md-3 col-sm-offset-1 col-sm-4 col-xs-12">
        <h4 class="title">Contact Us</h4>
        <div class="contact_wrapper sameheight3">
          <address>
          <figure class="icon"> <i class="fa fa-map-marker" aria-hidden="true"></i> </figure>
          <div class="details"> <span> Lalitpur,Kupondole<br>
          Kathmandu </span> </div>
          </address>
          <address>
          <figure class="icon"> <i class="fa fa-phone" aria-hidden="true"></i> </figure>
          <div class="details"> <span> <a href="tel:015555039" class="linkto">015555039 </a> </span> </div>
          </address>
          <address>
          <figure class="icon"> <i class="fa fa-envelope" aria-hidden="true"></i> </figure>
          <div class="details"> <span> <a href="mailto:info@goldentriangle.com.np" class="linkto">info@goldentriangle.com.np</a> </span> </div>
          </address>
        </div>
      </div>
      <div class="col-md-offset-1 col-md-3 col-sm-4 col-xs-12">
        <h4 class="title">Get Social</h4>
        <ul class="social-icons sameheight3">
          <li><a href="#" data-toggle="tooltip" data-placement="top" title="" data-original-title="Facebook"><i class="fa fa-facebook" aria-hidden="true"></i> </a></li>
          <li><a href="#" data-toggle="tooltip" data-placement="top" title="" data-original-title="Twitter"><i class="fa fa-twitter" aria-hidden="true"></i> </a></li>
          <li><a href="#" data-toggle="tooltip" data-placement="top" title="" data-original-title="Linkedin"><i class="fa fa-linkedin" aria-hidden="true"> </i></a></li>
          <li><a href="#" data-toggle="tooltip" data-placement="top" title="" data-original-title="Google-plus"><i class="fa fa-google-plus" aria-hidden="true"></i> </a></li>
        </ul>
      </div>
    </div>
    <div class="copyright">
      <p>© Copyright © 2018. Golden Triangle. All Right Reserved.</p>
    </div>
  </div>
</section>
<section class="back-top">
  <div id="back-top" class="uparrow" style="display: block;"> <a href="#top" title="Go to the top of the page"> <i class="fa fa-angle-up"></i> </a> </div>
</section>
</body>
<script src="js/jquery-1.9.1.min.js"></script>
<!--./testimonial slider starts-->
 
 
<!--./testimonial slider ends-->
<script src="js/fixed-nav.js"></script>
<script src="js/jquery.js"></script>
<script src="js/bootstrap.js"></script>
<script src="js/Push-up.js"></script>
 

<!--..//SAME HEIGHT JQUERY STARTS-->
<script type="text/javascript" src="js/sameheight.js">
</script>

<!--..//SOCIAL ICON HOVER TOOLTIP STARTS-->
<script>
$(document).ready(function(){
    $('[data-toggle="tooltip"]').tooltip();   
});
</script>
</html>
